### PR TITLE
Fix missing error check of bufio.Scanner

### PR DESCRIPTION
**Pull request by forsaken628**
maybe more

### CODE REVIEW

**Review by KN4CK3R — COMMENTED** [review]
Does `scan: %w` help with debugging?

**Comment by forsaken628** [comment]
https://pkg.go.dev/fmt@go1.22.1#Errorf
yes

**Comment by KN4CK3R** [comment]
I know what `%w` does but I don't think `scan: ` adds some value.

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Review by lunny — APPROVED** [review]

File: models/asymkey/ssh_key_authorized_keys.go [hunk 1]
```diff
@@ -152,6 +152,10 @@ func RegeneratePublicKeys(ctx context.Context, t io.StringWriter) error {
 				return err
 			}
 		}
+		err = scanner.Err()
+		if err != nil {
+			return fmt.Errorf("scan: %w", err)
```
Comment (wxiaoguang) [inline models/asymkey/ssh_key_authorized_keys.go:157]: If it returns here, the `f.Close` call is missing? Maybe resource leaking.

(And there seem to be more similar cases below)

Comment (forsaken628) [reply models/asymkey/ssh_key_authorized_keys.go:157]: I agree, but this PR was merged.

Comment (wxiaoguang) [reply models/asymkey/ssh_key_authorized_keys.go:157]: ->  Fix some pending problems #29985

**Pull request merged**